src/app/shared/services/layout.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { delay } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LayoutService {
+  // Service allowing modifications of the layout
+
+  private leftMenuAndHeadersDisplayed = new BehaviorSubject<boolean>(true);
+  leftMenuAndHeadersDisplayed$ = this.leftMenuAndHeadersDisplayed.asObservable().pipe(delay(0));
+
+  private withTask = new BehaviorSubject<boolean>(false);
+  withTask$ = this.withTask.asObservable().pipe(delay(0));
+
+  toggleLeftMenuAndHeaders(shown?: boolean): void {
+    if (shown !== undefined) {
+      this.leftMenuAndHeadersDisplayed.next(shown);
+    } else {
+      this.leftMenuAndHeadersDisplayed.next(!this.leftMenuAndHeadersDisplayed.value);
+    }
+  }
+
+  toggleWithTask(newState: boolean): void {
```

Comment:
As you do not use it in the PR, I do not know what you want to do with it exactly but it looks a bit strange to have to independent observable for things which are not independent.

Reply:
Added usage of withTask$ : removes padding-bottom in the right side of app.component.html when a task is loaded, so the task can be displayed with all available area.